inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -401,6 +401,14 @@ protected function get_first_links( $post_type ) {
 			$front_page['chf'] = 'daily';
 			$front_page['pri'] = 1;
 
+			$images = [];
+			/**
```

Comment:
```suggestion
			/**
			 * Filter images to be included for the term in XML sitemap.
			 *
			 * @param array $images The list of images to include in the XML sitemap.
			 * @return array $images  Array of image items.
			 */
			 $image_list = apply_filters( 'wpseo_sitemap_urlimages_front_page', $images );

			if ( isset( $image_list ) && is_array( $image_list ) ) {
				$front_page['images'] = $image_list;
			}
		
```

Reply:
Added this suggestion how we could add an additional check to make sure the images key can only have an array, if you now return an int or a string in the filter your sitemap can break.

Reply:
Good point, I'm adding it also to the other sitemap url images filters